packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -295,6 +296,13 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "binance", pair: "btcusdt" },
       { type: "cryptowatch", exchange: "bitstamp", pair: "btcusd" }
     ]
+  },
+  USDPERL: {
```

Comment:
Adds the USDPERL price feed to enable the identifier to be queried to get historic prices. For now, this is simply in this file but in future, these configs should be refactored out to a separate file. I created an issue to track this: https://github.com/UMAprotocol/protocol/issues/2154